server/app/mutations/volumes/plugin_install.rb

```diff
@@ -0,0 +1,41 @@
+module Volumes
+  class PluginInstall < Mutations::Command
+
+
+    required do
+      model :grid, class: Grid
+      string :name
+    end
+
+    optional do
+      string :alias_name
+      array :config do
+        string
+      end
+      string :label
+    end
+
+    def validate
+
+    end
+
+    def execute
+      # TODO
```

Comment:
Why are we sending rpc calls here? Imho more robust way is to pass plugin information to agent via agent pull (node info maybe?)